spec/syntax.md

```diff
@@ -331,25 +394,45 @@ This serves 3 purposes:
 pattern = "{" *(text / expression) "}"
 ```
 
-> Example:
->
+> **Example**
+> 
+> A simple _pattern_ containing _text_:
 > ```
 > {Hello, world!}
 > ```
-
-Whitespace within a _pattern_ is meaningful and MUST be preserved.
+>
+> An empty _pattern_:
+> ```
+> {}
+> ```
+>
+> Some _patterns_ with _expressions_:
+> ```
+> {{$foo}}
+> {Hello {$user}!}
+> {You sent {$count :number maxFractionDigits=0} notifications to {$numFriends :number type=spellout} friends.}
+> ```
+>
+> A _pattern_ containing three spaces:
+> ```
+> {   }
+> ```
 
 ### Expressions
 
-**_Expressions_** MUST start with an _operand_ or an _annotation_.
+An _expression_ can appear as a local variable value, as a _selector_, and within a _pattern_.
+Each _expression_ MUST start with an _operand_ or an _annotation_.
 An _expression_ MUST NOT be empty.
 
-An **_operand_** is either a _literal_ or a _variable_.
+An **_operand_** is a _literal_ or a _variable_ to be evaluated in an _expression_.
```

Comment:
```suggestion
An **_operand_** MUST appear in an _expression_. An _operand_ is either a _literal_ or a _variable_.
```

Reply:
Your proposed change loses the "definition form" that this PR is adding, that is, that a definition say *what* something is before imposing anything on it.

> An **_operand_** is _whatever it is_.

It can then go on to say that it MUST (only) appear in an expression (note that your phrase is misleading, since an expression  is not required to have an operand)